sig-governance.md

```diff
@@ -0,0 +1,179 @@
+repository.
+- Record meetings and make them publicly available.
+- Report activity with the community via the [contributors@lists.opendatahub.io](mailto:contributors@lists.opendatahub.io) mailing list at
+least once a year. 
+  - Each SIG is assigned an update during the [Open Data Hub community meeting](./README.md) throughout the year.
+- Participate in release planning meetings and retrospectives, and burndown
+meetings, as needed.
+- Ensure related work happens in a project-owned github repository, with
+ code and tests explicitly owned and supported by the SIG, including issue
+ triage, PR reviews, test-failure response, bug fixes, etc.
+- Use the [forums provided](./README.md) as the primary means of working, communicating, and
+collaborating, as opposed to private emails and meetings.
+- Ensure contributing instructions are defined in the SIGs
+folder located in the opendatahub-io/opendatahub-community repository if the groups contributor steps
+and experience are different or more in-depth than the documentation listed in
+the general [contributor guide](./contributing.md).  
+- Help and sponsor working groups in which the SIG has interest.  
+- Track and identify all SIG features in the current release.
+
+## Roles
+
+### Notes on Roles
+
+Within this section "Lead" refers to someone who is a member of the union
+ of a Chair or Subproject Owner role. There is no one lead to any
+ Open Data Hub community group. Leads have specific decision making power over some
+ part of a group and thus additional accountability. Each role is detailed below.  
+
+- Initial roles are defined at the founding of the SIG or Subproject as part
+of the acceptance of that SIG or Subproject.
+
+#### Activity Expectations  
+
+- Leads *SHOULD* remain active and responsive in their Roles.
+- Leads taking an extended leave of 1 or more months *SHOULD* coordinate with other leads to ensure the role is adequately staffed during the leave.
+- Leads going on leave for 1-3 months *MAY* work with other Leads to identify a temporary replacement.
+- Leads of a role *SHOULD* remove any other leads or roles that have not communicated a leave of absence and either cannot be reached for more than 1 month or are not fulfilling their documented responsibilities for more than 1 month.
+  - This may be done through a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) vote of Leads. If there are not enough *active* Leads, then a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) vote between Chairs and Subproject Owners may decide the removal of the Lead.
+
+#### Requirements
+
+- Leads *MUST* be at least a ["member" on our contributor ladder](./community-membership.md) to
+be eligible to hold a leadership role within a SIG.
+- SIGs *MAY* prefer various levels of domain knowledge depending on the
+role. This should be documented.  
+- People management interests.
```

Comment:
Can you elaborate on this point?

Reply:
The essence of this is that running a SIG will require assisting with managing people's time and contributions.  If one is not interested in working with people in this capacity, they should not elect to be a lead.  Does that make sense?  I can attempt to clarify this line a bit more as it does seem to need it.

Reply:
That makes sense! I'd definitely elaborate a little bit.